code/_onclick/click.dm

```diff
@@ -93,7 +93,7 @@
 
 		return
 
-	if(!isturf(loc) && !is_holder_of(src, A)) // Can't touch anything from inside a locker/sleeper etc, unless it's inside our inventory.
+	if(A != loc && !isturf(loc) && !is_holder_of(src, A)) // Can't touch anything from inside a locker/sleeper etc, unless it's inside our inventory.
```

Comment:
I did it as bauds said - can attack_hand your location, but can't use items on it